__tests__/endpoints/topics.test.js

```diff
@@ -0,0 +1,24 @@
+const { app, data, seed, db, request } = require("../../testImports");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+	test("GET: 200 sends an array of topics to the client", () => {
+		return request(app)
+			.get("/api/topics?sort")
+			.expect(200)
+			.then(({ body: { topics } }) => {
+				expect(topics.length).toBe(3);
+				topics.forEach((topic) => {
+					expect(typeof topic.slug).toBe("string");
+					expect(typeof topic.description).toBe("string");
+				});
+			});
+	});
+  test("GET: 404 Returns an error if incorrect path is used", () => {
```

Comment:
Solid test, should technically not be in the describe block for /api/topics, as this could apply to any bad endpoint like `/api/notAnEndpoint`